server/app.ts

```diff
@@ -59,16 +60,21 @@ app.use(express.json()); // parse application/json
 
 // routers
 app.use('/countries', countriesRouter);
+app.use('/v2/countries', countriesRouter);
 app.use('/trees', treesRouter);
 app.use('/planters', plantersRouter);
 app.use('/organizations', organizationsRouter);
+app.use('/v2/organizations', organizationsRouterV2);
 app.use('/species', speciesRouter);
+app.use('/v2/species', speciesRouter);
 app.use('/wallets', walletsRouter);
+app.use('/v2/wallets', walletsRouter);
 app.use('/transactions', transactionsRouter);
 app.use('/tokens', tokensRouter);
+app.use('/v2/tokens', tokensRouter);
 app.use('/v2/captures', capturesRouter);
 app.use('/raw-captures', rawCapturesRouter);
-app.use('/grower-accounts', growerAccountsRouter);
+app.use('/v2/growers', growerAccountsRouter);
 app.use('/v2/trees', treesRouterV2);
 app.use('/bounds', boundsRouter);
```

Comment:
@jeremydthomas I'm confused by this, is this your code for the v2? Why it shows up here?

Reply:
@dadiorchen that is a good question, I have no idea.

Reply:
@dayaneoli could you do some clean up on this PR? IMO, we can checkout the main branch and `git cherry-pick` your commit (valid part to solve the problem) then raise a new PR?

Reply:
I forkei the repository and put the object my set(). In the rest of the code I didn't change anything.

Reply:
I'll try to do that and then a new PR. Thanks.